### DISCUSSION TITLE
Issue in request-transformers plugin

### DISCUSSION THREAD

**Discussion by kollol-chowdhury-incontact** [Kong/kong#8277] [Help]
when using request-transformers plugin,getting an error whenever there is \x09 (tab character) in header.
Something similar to this issue: https://github.com/openresty/openresty/issues/618.

Has anyone seen this issue before?

I am using kong 2.7 .

TIA

**Comment by kikito** [comment]
Hi, thanks for reporting this. Could you please share the error you get (what your logs say)?

**Reply by kollol-chowdhury-incontact** [reply]
Hi @kikito .. Thanks for replying back.I am attaching a screenshot of the error i am getting.

![image](https://user-images.githubusercontent.com/94824723/148930527-310c9ce7-0588-4aca-9b80-f7cfbe33c9bc.png)

**Reply by kollol-chowdhury-incontact** [reply]
Hi @kikito : Any update on this
